content/swift/concepts/protocols/terms/decode/decode.md

```diff
@@ -0,0 +1,93 @@
+---
+Title: 'Decoder'
+Description: 'Enables values to be decoded from a native format to a custom format.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Protocols'
+Catalog Content:
+  - 'learn-swift'
+  - 'paths/ios-developer'
+---
+
+**`Decoder`** is a type that allows the decoding of values from a native format such as JSON or XML  into an in-memory representation like a struct, class, or enum.
+
+> **Note:** If you are also looking to encode values into a native representation, it would be beneficial to use the `Codable` protocol as it supports the use of both `Encodable` and `Decodable`.
```

Comment:
```suggestion
> **Note:** If a given use-case requires encoding values into a native representation, it would be beneficial to use the `Codable` protocol as it supports the use of both `Encodable` and `Decodable`.
```

Reply:
first and second personal pronouns (I, you, we) are avoided generally in docs